android/app/build.gradle

```diff
@@ -134,15 +110,8 @@ def jscFlavor = 'org.webkit:android-jsc:+'
  * on project.ext.react, JavaScript will not be compiled to Hermes Bytecode
  * and the benefits of using Hermes will therefore be sharply reduced.
  */
-def enableHermes = project.ext.react.get("enableHermes", false);
+def enableHermes = hermesEnabled.toBoolean();
```

Comment:
What does this do?

Reply:
sets it to whatever was set in `gradle.properties`, in our case `false`
according to new template : https://github.com/facebook/react-native/blob/0.72-stable/packages/react-native/template/android/gradle.properties#L44